pandas/_libs/tslibs/period.pyx

```diff
@@ -1246,6 +1246,37 @@ cdef class _Period(object):
 
     @property
     def dayofweek(self):
+        """
+        Return the day of the week.
+
+        This attribute returns the day of the week on which the particular
+        date occurs with Monady=0, Sunday=6.
+
+        Returns
+        -------
+        Int:Range of 0 to 6
```

Comment:
Not following the style guide, see https://python-sprints.github.io/pandas/guide/pandas_docstring.html#section-4-returns-or-yields